jenkins-jobs/scripts/check-contributors.sh

```diff
@@ -1,30 +1,48 @@
 #!/bin/bash
 
+DIR="$HOME/debezium-copyright-list"
 COPYRIGHT="COPYRIGHT.txt"
-CONTRIBUTOR_NAMES="/tmp/repository-CONTRIBUTOR-NAMES.txt"
-CONTRIBUTORS="/tmp/repository-CONTRIBUTORS.txt"
-
 FILTERS="jenkins-jobs/scripts/config/FilteredNames.txt"
 ALIASES="jenkins-jobs/scripts/config/Aliases.txt"
 
-git log --pretty=format:"%an" . | sort | uniq > $CONTRIBUTOR_NAMES
-git log --pretty=format:"%an|%ae" . | sort | uniq > $CONTRIBUTORS
+mkdir -p $DIR
+
+CONTRIBUTOR_NAMES="$DIR/repository-CONTRIBUTOR-NAMES.txt"
+CONTRIBUTORS="$DIR/repository-CONTRIBUTORS.txt"
+COPYRIGHT_ALIASES="$DIR/Aliases.txt"
+COPYRIGHT_FILTERS="$DIR/FilteredNames.txt"
+COPYRIGHT_DEBEZIUM_REPOS="$DIR/COPYRIGHT.txt"
+
+cp $ALIASES $FILTERS $COPYRIGHT $DIR && cd $DIR
+
+declare -a DEBEZIUM_REPOS=("debezium" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "docker-images")
```

Comment:
Would it be possible to add the repo list as cmd line params and if not provided set it to this default?

Reply:
Yes, a list or series of arguments can be passed. But how are they getting checkout? It would be great if you could share an example.

Reply:
Please see above